data/pilots/rebel-alliance/yt-2400-light-freighter.json

```diff
@@ -119,6 +131,123 @@
       "extended": true,
       "keywords": ["Freighter"],
       "epic": true
+    },
+    {
+      "name": "“Leebo”",
+      "caption": "Wisdom of Ages",
+      "initiative": 3,
+      "limited": 1,
+      "cost": 6,
+      "loadout": 16,
+      "xws": "leebo-swz103",
+      "ability": "After you defend or perform an attack, if you spent a calculate token, gain 1 calculate token.",
+      "shipActions": [
+        { "difficulty": "White", "type": "Calculate" },
+        {
+          "difficulty": "White",
+          "linked": { "difficulty": "Red", "type": "Rotate Arc" },
+          "type": "Lock"
+        },
+        { "difficulty": "Red", "type": "Barrel Roll" },
+        { "difficulty": "White", "type": "Rotate Arc" }
+      ],
+      "shipAbility": {
+        "name": "Sensor Blindspot",
+        "text": "While you perform a primary attack at attack range 0-1, you roll 1 less attack die. While you defend at range 1, you roll 1 less defense die."
```

Comment:
I kept the stupid leebo typo. I doubt they'll fix it, and I doubt it matters, but noting in case anyone cares, haha.